### ISSUE TITLE
Complete ReadMe File

### ISSUE THREAD

**Issue by 3osmic** [3osmic/Parking-Lot-System#1]
This file needs to completed as we go through the project because it consists of an in-depth description of what the project does.

**Comment by CaseeNewton** [comment]
We will be using a parking lot at Wake Tech to help plan and design our QR code parking system. We have discussed the MOSCOW on what we absolutly will need from this project. The 'must' are set in stone it seems but we still have some flexability on the rest. Hoping to solidify in our week 3 meeting the specifics on what the parking lot system will do/ won't do. As of now the system will be an app to download for anyone visiting waketech campus, and will be easily scannable for those in a hurry and accessable for those with disabilities. This app will save your car's approximate location to assist in findind your way back to it from any location.

**Comment by 3osmic** [comment]
I will be adding all info here to the readme file in the repository.

**Comment by CaseeNewton** [comment]
Draft overview submitted for week 3, document has been submitted in blackboard.

**Comment by CaseeNewton** [comment]
This week we will meet Tues & Thursday in Teams to go over and complete the SRS documentation. This will be our first draft submission of the V1 of the SRS. After revision and collaboration we will submit for review. We will be giving a more detailed outlook on what the final version of what we will accomplish with our Parking Lot Project. Each member has been assigned a specific task this week by our group leader and will be expected to complete on their own or with assistance if asked. We will go over this in our Thursday meeting to fine tune the SRS document.

**Comment by CaseeNewton** [comment]
Met up week of 10/22-10/29 for the SRS V2 of the first draft of High level design on the Parking Lot project. We equally divided material between the group. We submitted the draft of V2 and are currently waiting on a peer review from our mentor. With the review from our mentor we will meet this week and work on the SRS V2 document to turn in for a finalized version. We will revise and complete it by the end of this week for submission.
